scripts/utils/generate_package_wiki.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import re
+import datetime
+import argparse
+
+
+packagesByCategory={}
```

Comment:
Follow [variables naming convention in Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#316-naming)
```suggestion
packages_by_category = {}
```